providers/google/src/airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -86,6 +89,11 @@
     from google.api_core.retry import Retry
     from requests import Session
 
+    if AIRFLOW_V_3_0_PLUS:
+        from airflow.sdk.definitions.context import Context
+    else:
+        from airflow.utils.context import Context
```

Comment:
This is not needed. The type-checking block can always import from `airflow.sdk`.

```suggestion
    from airflow.sdk import Context
```